src/apps/jewelswap/index.tsx

```diff
@@ -0,0 +1,191 @@
+import { Box, Grid, TextField, useMediaQuery } from '@mui/material';
+import BigNumber from '@multiversx/sdk-core/node_modules/bignumber.js';
+import React, { useCallback, useMemo, useState } from 'react';
+import { useOrganizationInfoContext } from 'src/pages/Organization/OrganizationInfoContextProvider';
+import { Text } from 'src/components/StyledComponents/StyledComponents';
+import { MainButton } from 'src/components/Theme/StyledComponents';
+// import { PropertyKeyBox } from 'src/components/Utils/PropertKeyBox';
+import { useNftAuctionClaimableAmount } from 'src/utils/useNftAuctionClaimableAmount';
+import { Address, BigUIntValue } from '@multiversx/sdk-core/out';
+import { mutateSmartContractCall } from 'src/contracts/MultisigContract';
+import { jewelSwapLendingContractAddress } from 'src/config';
+import { currentMultisigContractSelector } from 'src/redux/selectors/multisigContractsSelectors';
+import { useSelector } from 'react-redux';
+import { inputLabelClasses } from '@mui/material/InputLabel';
+import { theme } from '@components/Theme/createTheme';
+import { MultiversXLogo } from '@components/Utils/MultiversXLogo';
+import { NFTMarketplace } from './types';
+import { marketplaces } from './constants';
+import * as Styled from './styled';
+
+const ClaimNftAuction = () => {
+  const claimableAmountResult = useNftAuctionClaimableAmount();
```

Comment:
Do you use this or is it left behind from our app? We should delete it in this component, if possible. Thanks!